gradle.properties

```
@@ -0,0 +1,11 @@
+org.gradle.parallel=true
+kotlin.parallel.tasks.in.project=true
+kotlin.code.style=official
+
+# Library Setting
+group=be.zvz.koreanbots
+version=1.0.0
```

```suggestion
version=2.0.0
```
기존 `v1`과 호환되지 않는 변경 사항이 있기 때문에, major 버전을 변경하는 것이 좋아보입니다.

---

다른 변경 사항 적용 후 반영하겠습니다.